ygen/testdata/structs/openconfig-list-enum-key.getters-append.formatted-txt

```diff
@@ -382,6 +394,15 @@ func (t *OpenconfigListEnumKey_Top_SingleKey) GetEks(K E_OpenconfigListEnumKey_T
   return nil
 }
 
+// DeleteEks deletes the value with the specified keys from
+// the receiver OpenconfigListEnumKey_Top_SingleKey.
+func (t *OpenconfigListEnumKey_Top_SingleKey) DeleteEks(K E_OpenconfigListEnumKey_Top_SingleKey_Eks_Config_K) {
+
```

Comment:
(nit, optional) Do you want to consider removing the additional whitespace at the start of the function here?

Reply:
Done.